src/scss/components/_scenarios.scss

```
@@ -1,7 +1,15 @@
+* {
+  margin: 0 auto;
+  max-width: 450px;
+}
 .scenarios {
   padding-bottom: 230px;
+  padding-left: 30px;
+  padding-right: 30px;
   padding-top: 30px;
```

You can write this 4 properties using the short-hand property, like so:
```suggestion
  padding: 30px 30px 230px;
```

---

Done. Changed!